src/program.fs

```diff
@@ -80,7 +80,7 @@ module Program =
     /// Trace all the messages as they update the model
     let withTrace trace (program: Program<'arg, 'model, 'msg, 'view>) =
         { program
-            with update = fun msg model -> trace msg model; program.update msg model}
+            with update = fun model msg -> trace msg model; program.update model msg}
```

Comment:
I'm not sure it's relevant to invert the args here, it's a question of consistency (keeping the args always in the same order), versus backward compatibility.

Reply:
you mean invert them for the `trace` call yeah that's debatable. But might as well change them, breaking for breaking...

Reply:
(But it's @et1975  call 😄 )